src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -92,8 +92,9 @@ private sealed partial class UIThreadOnly : IDisposable
 
             private readonly IWaitIndicator _waitIndicator;
 
-            public readonly ITextBuffer OutputBuffer;
-            public readonly ITextBuffer StandardInputBuffer;
+            public ITextBuffer OutputBuffer { get; }
+            public ITextBuffer HistoryBuffer { get; private set; }
```

Comment:
I'm not sure I understand why do we need a new buffer for this. Why can't use use AddInput(string)?